README_LINUX.md

```diff
@@ -88,155 +79,248 @@ sudo apt-get install \
   libavahi-client-dev \
   libudev-dev
 ```
-If you need to use JACK1 replace libjack-jackd2-dev with libjack-dev.
 
-Installing requirements on Fedora
----------------------------------
-
-The following commands should install all the recommended SuperCollider dependencies on Fedora, except for Qt:
-```shell
-sudo dnf groupinstall "Development Tools"
-sudo dnf install cmake libsndfile-devel wayland-devel xorg-x11-server-Xwayland-devel pipewire-devel pipewire-jack-audio-connection-kit-devel systemd-devel fftw-devel alsa-lib-devel libatomic
-sudo dnf install emacs # if building with the sc-el backend (default)
-sudo dnf install libXt-devel 
-```
+If you need to use JACK1 replace `libjack-jackd2-dev` with `libjack-dev`.
 
-Installing Qt
--------------
+#### Installing Qt
 
 **Qt 6.2 or later** is required to be able to run the SuperCollider IDE and sclang's Qt GUI system.
 
-### Installing Qt on recent Debian-like operating systems
-
 Try this command to query the Qt6 version available to you:
+
 ```shell
 apt-cache policy qt6-base-dev
 ```
-If this displays version 6.2 or later, you can install Qt6 using `apt`. Please note that this list of packages might not be complete, as it hasn't been revised for Qt6.
+
+If this displays version 6.2 or later, you can install Qt6 using `apt-get`.
+Please note that this list of packages might not be complete, as it hasn't been revised for Qt6.
+
 ```shell
-sudo apt-get install qt6-base-dev qt6-base-dev-tools qt6-tools-dev qt6-tools-dev-tools libqt6websockets6-dev libqt6webenginecore6 qt6-webengine-dev qt6-webengine-dev-tools libqt6svgwidgets6
+sudo apt-get install \
+  qt6-base-dev \
+  qt6-base-dev-tools \
+  qt6-tools-dev \
+  qt6-tools-dev-tools \
+  libqt6websockets6-dev \
+  libqt6webenginecore6 \
+  qt6-webengine-dev \
+  qt6-webengine-dev-tools \
+  libqt6svgwidgets6
```

Comment:
I personally prefer everything in a single line... but I guess it's fine with escaped newlines.
Is it really better for readability or usability to have them listed in separate lines?

Reply:
IMO its easier to skim over the list to get an understanding what gets installed - it also allows to annotate specific dependencies - e.g. we could comment that it is possible to build w/o webengine.